charts/signoz/values.yaml

```
@@ -554,6 +554,12 @@ frontend:
     #    hosts:
     #      - frontend.domain.com
 
+  # -- Nginx congifmap customize for new parameters
+  customConfig: false
+  # customConfig: true
+  # data: |
+  #   large_client_header_buffers 4 256k;
+
```

We can also set it by default. We include the following configurations in Docker Standalone, but missed out
on Swarm and K8s.

https://github.com/SigNoz/signoz/blob/develop/deploy/docker/common/nginx-config.conf#L14-L17

```suggestion
  # -- Frontend Nginx extra configurations
  nginxExtraConfig:
      client_max_body_size 24M;
      large_client_header_buffers 8 16k;
```

---

It would be better to set it by default if it is already used in Docker standalone. Let me do the changes and test it on my local.